index.js

```diff
@@ -246,6 +246,22 @@ export async function setupPlugin({ config, global, jobs }) {
     })
 }
 
+function getElementByOrderZero(json) {
+  //console.log("here ok");
```

Comment:
Commented log function - we should remove this as it is confusing for future readers
```suggestion
```